plugins/buffer/README.md

```diff
@@ -41,7 +41,7 @@ gets transferred to the destination).
 
 <div>
 
-[![](/images/fluentd-v0.14-plugin-api-overview.png)](/images/fluentd-v0.14-plugin-api-overview.png)
+[![](/images/fluentd-v0.14-plugin-api-overview.png)](https://github.com/fluent/fluentd-docs-gitbook/blob/1.0/images/fluentd-v0.14-plugin-api-overview.png)
```

Comment:
Is`![](/images/fluentd-v0.14-plugin-api-overview.png)` correct?

Reply:
actually I don't understand why  this format `[![x](y)](y)`, 
can't this `![x](y)` work?

Reply:
`[![x](y)](y)` means an image with a link like `<a href="/path1"><img src="/path2"></a>`.